6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<p>Make the header here something like "Model Comparison with WAIC" so it's  slightly more clear that it relates to model comparison.</p><p><br></p><p>Also, could you add a note that this step is likely to be temporary until MCMCChains supports these functions natively?</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Changed the header.

Added these lines: "The code block below manually calculates the WAIC. This is likely to be temporary until `MCMCChains` supports these functions natively."